packages/coinstac-graphql/src/data/resolvers.js

```diff
@@ -137,7 +137,15 @@ const resolvers = {
       return new Promise();
     },
     deleteConsortiumById: (_, args) => {
-      return new Promise();
+      return helperFunctions.getRethinkConnection()
```

Comment:
We should be watching permissions in our resolvers now, so I'd add a few lines here to limit deletion to only those with write access to the `consortia` table or write access to the submitted consortium id. Something like this:

```
deleteConsortiumById: ({ auth: { credentials: { permissions } } }, args) => {
      if (!permissions.consortia.write
          && args.consortium.id
          && !permissions.consortia[args.consortium.id].write) {
            return Boom.forbidden('Action not permitted');
      }

      return helperFunctions.getRethinkConnection()
      ...